### ISSUE TITLE
Is there no way to not transform idents which are also keywords?

### ISSUE THREAD

**Issue by wucke13** [holygits/casey#3]
I have a use case where I need to transform something which includes an `impl` from `shouty` to `pascal`. Is there really no way of making that transformation?

**Comment by jordy25519** [comment]
Hey @wucke13.
Looks like there were some holes in the pascal! case transformation
should be closed by #4 and available in the [0.3.1](https://crates.io/crates/casey/0.3.1) patch release

**Comment by wucke13** [comment]
If I understand the patch correctly, that is not the issue that I meant. I meant to ask whether there isn't any way to explicitly ignore keywords in the transformation, such as `impl` or `struct`.

**Comment by jordy25519** [comment]
> ignore keywords in the transformation, such as impl or struct.

Ah, I see. Should be fixed and available in [0.3.2](https://crates.io/crates/casey/0.3.2)
```rust
pascal!(fn TEST_THIS {});

// before patch
Fn TestThis {}

// after patch
fn TestThis {}
```

It maybe worth making this a flag in future but for now ignoring keywords is the intended behaviour.
Let me know if I didn't understand this time :)

**Comment by wucke13** [comment]
Guessing from the tests, this fixes my issue. Really nice!
